charts/coredns/README.md

```diff
@@ -133,6 +133,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `autoscaler.livenessProbe.successThreshold`     | Minimum consecutive successes for the probe to be considered successful after having failed.                                              | `1`                                                         |
 | `deployment.enabled`                            | Optionally disable the main deployment and its respective resources.                                                                      | `true`                                                      |
 | `deployment.name`                               | Name of the deployment if `deployment.enabled` is true. Otherwise the name of an existing deployment for the autoscaler or HPA to target. | `""`                                                        |
+| `deployment.annotations`                        | Annotations to add to the main deployment                                                                                                 | `{}`                                                        |
```

Comment:
Can we add them to all objects as "customAnnotations" instead? Similar to how customLabels is done.

Reply:
I added customAnnotations additionally in case anyone needs it.
In my use case the annotations are specific to the deployment.
We use them to silence some warnings about a low user id used in the pod.
Its possible to use the customAnnotations  here, but that would somehow clutter the other resources